src/eve/concepts.py

```diff
@@ -233,53 +245,56 @@ class Config(FrozenModelConfig):
         pass
 
 
-class UIDGenerator:
-    """Simple unique id generator using a counter."""
+AnyNodeClasses = (bool, bytes, int, float, str, IntEnum, StrEnum, Node, list, dict, set)
 
-    #: Constantly increasing counter for generation of unique ids
-    __counter = itertools.count(1)
 
-    @classmethod
-    def get_unique_id(cls, *, prefix: Optional[str] = None) -> str:
-        """Generate a new globally unique id (for the current session)."""
-        count = next(cls.__counter)
-        return f"{prefix}_{count}" if prefix else f"{count}"
+class VType(FrozenModel):
+
+    # VType fields
+    #: Unique name
+    name: Str
+
+    def __init__(self, name: str) -> None:
+        super().__init__(name=name)
+
+
+# class Module:
+#     # root
 
-    @classmethod
-    def reset(cls, start: int = 1) -> None:
-        """Reset generator."""
-        cls.__counter = itertools.count(start)
+
+# class Program:
```

Comment:
Maybe a named todo?

Reply:
Yeah, actually it's not even clear that is useful. I will remove it.